plot/indicator/cci.go

```diff
@@ -0,0 +1,49 @@
+package indicator
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/rodrigo-brito/ninjabot/model"
+	"github.com/rodrigo-brito/ninjabot/plot"
+
+	"github.com/markcheno/go-talib"
+)
+
+func Cci(period int, color string) plot.Indicator {
+	return &cci{
+		Period: period,
+		Color:  color,
+	}
+}
+
+type cci struct {
+	Period int
+	Color  string
+	Values model.Series
+	Time   []time.Time
+}
+
+func (c cci) Name() string {
+	return fmt.Sprintf("CCI(%d)", c.Period)
+}
+
+func (c cci) Overlay() bool {
+	return false
+}
+
+func (c *cci) Load(dataframe *model.Dataframe) {
+	c.Values = talib.Cci(dataframe.High, dataframe.Low, dataframe.Close, c.Period)
+	c.Time = dataframe.Time
```

Comment:
Can you remove the warmup perido?
```suggestion
	c.Time = dataframe.Time[c.Period:]
	c.Values = c.Values[c.Period:]
```

Reply:
Done, sorry overlooked it

Reply:
Done, sorry overlooked it